packages/web-app-admin-settings/src/mixins/general/resetLogo.ts

```diff
@@ -0,0 +1,48 @@
+import { mapActions } from 'vuex'
+import { clientService } from 'web-pkg'
+
+export default {
+  computed: {
+    $_resetLogo_items() {
+      return [
+        {
+          name: 'reset-logo',
+          icon: 'restart',
+          label: () => {
+            return this.$gettext('Reset logo')
+          },
+          handler: this.$_resetLogo_trigger,
+          componentType: 'button',
+          class: 'oc-general-actions-reset-logo-trigger'
+        }
+      ]
+    }
+  },
+  methods: {
+    ...mapActions(['showMessage']),
+
+    $_resetLogo_trigger() {
+      this.$_resetLogo_reset()
+    },
+
+    async $_resetLogo_reset() {
+      try {
+        const accessToken = this.$store.getters['runtime/auth/accessToken']
+        const httpClient = clientService.httpAuthenticated(accessToken)
+        await httpClient.delete('/branding/logo')
+        this.showMessage({
+          title: this.$gettext('Logo was reset successfully')
```

Comment:
```suggestion
          title: this.$gettext('Logo was reset successfully. Reloading page...')
```
We should announce the page reload